### PR TITLE
added dependency for lof plugin

### PR DESCRIPTION
**Pull request by matthdsm**
* [x] I have read the guidelines above.
* [ ] This PR adds a new recipe.
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/variant-effect-predictor/build.sh [hunk 1]
```diff
@@ -29,3 +29,4 @@ rm -rf t/
 wget --no-check-certificate https://raw.githubusercontent.com/ensembl-variation/VEP_plugins/master/CADD.pm
 wget --no-check-certificate https://raw.githubusercontent.com/ensembl-variation/VEP_plugins/master/dbNSFP.pm
 wget --no-check-certificate https://raw.githubusercontent.com/konradjk/loftee/master/LoF.pm
```
Comment (bgruening) [inline recipes/variant-effect-predictor/build.sh]: Can we use here a specific revision for all scripts or modules?

Comment (matthdsm) [inline recipes/variant-effect-predictor/build.sh]: I don't really think so, the ensembl-variation repo only has a specific branch for the latest  version of vep.
the loftee repo does have some version tags, but the last tag was made over a year ago, while production ready changes still get pushed to the master branch

Comment (bgruening) [inline recipes/variant-effect-predictor/build.sh]: You could use such a link isn't it: https://raw.githubusercontent.com/Ensembl/VEP_plugins/435d16729a518a83b83787985987f8f2c2782e2c/dbNSFP.pm

Comment (matthdsm) [inline recipes/variant-effect-predictor/build.sh]: true, but that doesn't really relate to some version does it? This just fixes to plugin to some arbitrary point in the history of the repo. How would we pinpoint the right commit?

Comment (bgruening) [inline recipes/variant-effect-predictor/build.sh]: What is the `right commit`? The problem is that you don't know what you get with the current approach. You will get the script from the time of build. If you rebuild the package you can get an entire new script in worst case. With fixing a revision you are guaranteed to at least get every time the same script.

Comment (matthdsm) [inline recipes/variant-effect-predictor/build.sh]: Ok, that's also a valid point. 
I'll just point the url's to to the latest commit on the master branch. The plugin url's can then be updated together with the vep version.

Comment (bgruening) [inline recipes/variant-effect-predictor/build.sh]: Thansk!

**Comment by matthdsm** [comment]
@bgruening 
Checks failed because zlib.h library header is missing. What do you propose I do about this? Do I create a new vep recipe? of is this something that's just missing in the Travis platform?

**Comment by bgruening** [comment]
Depend on `- zlib` as build and runtime dependency. It can be that you need to add `export C_INCLUDE_PATH=$PREFIX/include` to your build.sh script as well.

Thanks!

**Comment by matthdsm** [comment]
Ok, so I've added another sed statement to change the hardcoded path of the zlib library. we'll see how this goes

**Comment by matthdsm** [comment]
Huraah! This seems to have fixed it!

**Comment by bgruening** [comment]
Great work! Thanks!

**Comment by matthdsm** [comment]
yay!

**Pull request merged**